lib/pos_blockchain.go

```diff
@@ -1860,6 +1879,12 @@ func (bc *Blockchain) getUtxoViewAtBlockHash(blockHash BlockHash) (*UtxoView, er
 	}
 	// Update the TipHash saved on the UtxoView to the blockHash provided.
 	utxoView.TipHash = &blockHash
```

Comment:
Not a direct change in this PR, but shouldn't ConnectBlock update the tipHash for the view?

Reply:
it does! not sure why I had to add this here earlier. I'll take a look at it another time.